website/docs/dictformats.md

```diff
@@ -2,13 +2,13 @@ Popular dictionary formats are all supported.
 
 ## Local Dictionaries Sources
 
-* MDict dictionaries (.mdx/.mdd)
-* StarDict dictionaries (.ifo/.dict./.idx/.syn)
+* [MDict](https://www.mdict.cn/) dictionaries (.mdx/.mdd)
+* [StarDict](http://www.huzheng.org/stardict/) dictionaries (.ifo/.dict./.idx/.syn)
 * DSL dictionaries (ABBYY Lingvo source files .dsl(.dz))
```

Comment:
https://anatoly314.github.io/dsl-manual/#../index.html
dsl document

Reply:
It is not official, and the actual source is 

https://lingvoboard.ru/store/html/DSLReference_HTML/index.html

:sweat_smile:

Reply:
Added

Reply:
Added.